src/stores/UserMediaStore.js

```diff
@@ -1,9 +1,12 @@
 import Vue from 'vue';
+import { disableVideoStream, disableAudioStream } from '@/utils/mediaDevicesUtils';
 
 export default {
     state: {
         videoDeviceId: null,
         audioDeviceId: null,
+        audioActive: true,
```

Comment:
why true as default ?

Reply:
This is default true so it can try to get the audio and video stream. If this fails, it will become false.

Reply:
:+1: